docs/cloud/pouta/persistent-volumes.md

```diff
@@ -101,6 +111,19 @@ line tools:
 openstack volume create --description "<description>" --size <size> <name>
 ```
 
+!!! Warning "No äkköset in name or description allowed"
+    This is a know bug in the volume system. Volume creation will fail
+    if its name or description contains any non ASCII characters, this
+    includes äkköset and any non standard characters.
+
+    ```sh
+    $ openstack volume create matrice --size=1 --description='Déjà vu'
```

Comment:
Same comment `äkköset -> non-ASCII` in the title as before, especially if the example you are giving is the sentence `Déjà vu`

If the original example of the command line is in the form
```
openstack volume create --description "<description>" --size <size> <name>
```
then it might make it more easy to understand if also the "wrong" example follows the same order of arguments
```
openstack volume create --description='Déjà vu' --size=1 matrice
```

Reply:
In principle the "correct" way is:

```sh
openstack volume create --size <size> --description "<description>"  <name>
```

```sh
usage: openstack volume create [-h] [-f {json,shell,table,value,yaml}] [-c COLUMN] [--noindent] [--prefix PREFIX]
                               [--max-width <integer>] [--fit-width] [--print-empty] [--size <size>]
                               [--type <volume-type>]
                               [--image <image> | --snapshot <snapshot> | --source <volume>]
                               [--description <description>] [--availability-zone <availability-zone>]
                               [--consistency-group consistency-group>] [--property <key=value>] [--hint <key=value>]
                               [--bootable | --non-bootable] [--read-only | --read-write]
                               <name>
```

But I changed as you requested it.